src/audio/host.c

```diff
@@ -746,10 +746,10 @@ static int host_pointer_reset(struct comp_dev *dev)
 static int host_stop(struct comp_dev *dev)
 {
 	/* reset host side buffer pointers */
-	host_pointer_reset(dev);
+	//host_pointer_reset(dev);
```

Comment:
Have you verified this change? I think ALSA expects position to be reset after stop.